lib/config/config.js

```diff
@@ -167,6 +167,39 @@ module.exports = function (fs, path, url, convict) {
         'restmail.net',
         'mockmyid.com'
       ]
+    },
+    ipBlocklist: {
+      enable: {
+        doc: 'Flag to enable ip blocklist',
+        format: Boolean,
+        default: true,
+        env: 'IP_BLOCKLIST_ENABLE'
+      },
+      logOnly: {
+        doc: 'Flag to only log hits',
+        format: Boolean,
+        default: true,
+        env: 'IP_BLOCKLIST_LOGONLY'
+      },
+      lists: {
+        doc: 'A array of ip blocklist file paths',
+        format: Array,
+        default: [
+          './../test/mocks/firehol_level1.netset',
```

Comment:
Looks funny to have test mocks as the default in config. Seems they should either be in a test config, or the default config should have its files promoted out of 'mocks'.